src/_slugline_vars.scss

```diff
@@ -13,3 +13,20 @@ $link-color: black;
 // Navbar variables
 $navbar-light-color: $primary;
 $navbar-brand-font-size: 2rem;
+
+// Forms
+$input-border-radius: 0;
+$input-border-width: 1px;
+$input-border-color: black;
```

Comment:
Should be a variable.

(sidenote: I think black is too black, would prefer the original bootstrap colour)